colorPair.py

```diff
@@ -0,0 +1,39 @@
+MAJOR_COLORS = ['White', 'Red', 'Black', 'Yellow', 'Violet']
+MINOR_COLORS = ["Blue", "Orange", "Green", "Brown", "Slate"]
+
+
+def color_pair_to_string(major_color, minor_color):
+    return f'{major_color} {minor_color}'
+
+
+def get_color_from_pair_number(pair_number):
+    zero_based_pair_number = pair_number - 1
+    major_index = zero_based_pair_number // len(MAJOR_COLORS)
+    if major_index >= len(MAJOR_COLORS):
+        raise Exception('Major index out of range')
+    minor_index = zero_based_pair_number % len(MINOR_COLORS)
+    if minor_index >= len(MINOR_COLORS):
+        raise Exception('Minor index out of range')
+    return MAJOR_COLORS[major_index], MINOR_COLORS[minor_index]
+
+
+def get_pair_number_from_color(major_color, minor_color):
+    try:
+        major_index = MAJOR_COLORS.index(major_color)
+    except ValueError:
+        raise Exception('Major index out of range')
```

Comment:
Nice use of exception handling.
However, notice the error: The supplied color doesn't exist (does not have any index). The index isn't out of range.
Improving this message must be done in two places - one for major and one for minor colors. Refactor and change it in one place.